gnd/src/main/res/drawable/ic_signal_cellular_off_grey_700_24dp.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please remove temp code and assets used for local debugging.

Reply:
Done